crates/cdk-signatory/src/proto/signatory.proto

```diff
@@ -120,6 +131,28 @@ message BlindSignatureDLEQ {
   bytes s = 2;
 }
 
+// Witness type
+message Witness {
+  oneof witness_type {
+    P2PKWitness p2pk_witness = 1;
+    HTLCWitness htlc_witness = 2;
+  }
+}
+
+// P2PKWitness type
+message P2PKWitness {
+  // List of signatures
+  repeated string signatures = 1;
+}
+
+// HTLCWitness type
+message HTLCWitness {
+  // Preimage
+  string preimage = 1;
+  // List of signatures
+  repeated string signatures = 2;
+}
```

Comment:
Can't these be removed? The signatory does not verify HTLC or P2PK the mint does.